sofy_sofy/manifest.json

```diff
@@ -0,0 +1,73 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "eea6fdbc-2f8d-4483-bbd3-767818b1c25a",
+  "app_id": "sofy",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitoring the device metrics during the automated test case runs",
+    "title": "Sofy",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Connect SOFY with DataDog.",
+        "image_url": "images/datadog_connect.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Enable Application to send Metrics.",
+        "image_url": "images/datadog_monitoring.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Sofy Device Metrics.",
+        "image_url": "images/datadog_metrics.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "No Code Automated Testcase Test run.",
+        "image_url": "images/datadog_testperform.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Marketplace",
+      "Category::Monitoring",
+      "Offering::Integration",
+      "Submitted Data Type::Metrics"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "Sofy Overview": "assets/dashboards/sofy_sofy_overview.json"
+    },
+    "integration": {
+      "source_type_name": "Sofy",
+      "events": {
+        "creates_events": true
```

Comment:
Does this integration also send events to Datadog? If so, please include this under Data collected and in the overview